assets/src/application/ui/input/InlineEditInput/InlineEditText.tsx

```diff
@@ -1,10 +1,48 @@
 import React from 'react';
+import Dotdotdot from 'react-dotdotdot';
+
+import { Textfit } from 'react-textfit';
 
 import { TextProps } from './types';
-import { InlineEdit } from '@infraUI/inputs';
+import { InlineEdit, InlineEditPreviewProps } from '@infraUI/inputs';
+
+const Preview: React.FC<InlineEditPreviewProps> = ({ fitText, isEditing, onRequestEdit, value }) => {
+	if (isEditing) {
+		return null;
+	}
+
+	if (value.length > 50) {
```

Comment:
noticed that 50 char limit isn't fitting in one line:

![image](https://user-images.githubusercontent.com/1477453/81811083-bd954f00-952c-11ea-9cb9-41df55c280b5.png)

Reply:
changed to 40:

![image](https://user-images.githubusercontent.com/1477453/81811148-d30a7900-952c-11ea-878b-33b12c1da7f5.png)